uw-frame-components/portal/widgets/partials/compact-widget-card.html

```diff
@@ -0,0 +1,17 @@
+<md-card class="list-content" id="widget-id-{{::widget.nodeId}}">
+  <md-button class="widget-action widget-info md-icon-button" aria-label="see a brief description of this app">
```

Comment:
`aria-label` is good. As I understand it it would be better if the label included the name of the app that one would see a brief description of, so that screen reader user has more context in consuming that label.